main/keychain/src/main/java/org/cryptomator/keychain/LinuxKDEWalletKeychainAccessImpl.java

```diff
@@ -44,12 +44,12 @@ public void storePassphrase(String key, CharSequence passphrase) throws Keychain
                     !(wallet.hasEntry(handle, FOLDER_NAME, key, APP_NAME)
                             && wallet.entryType(handle, FOLDER_NAME, key, APP_NAME) == 1)
                     && wallet.writePassword(handle, FOLDER_NAME, key, passphrase.toString(), APP_NAME) == 0) {
-                log.debug("Passphrase successfully stored.");
+                LOG.debug("Passphrase successfully stored.");
             } else {
-                log.debug("Passphrase was not stored.");
+                LOG.debug("Passphrase was not stored.");
             }
         } catch (Exception e) {
-            log.error(e.toString(), e.getCause());
+            LOG.error(e.toString(), e.getCause());
```

Comment:
See my comment above. If the object is set up correctly, also here the stacktrace can be interesting.

Reconsider, if this should be changed to `LOG.error("Some msg:", e)`